shipitscript/src/shipitscript/data/create_new_release_task_schema.json

```diff
@@ -36,7 +36,7 @@
                 "cron_revision": {
                   "type": "string"
                 },
-                "repository_type": {
+                "repository_url": {
```

Comment:
I have a bit of a concern here with using an arbitrary repo url coming from the payload. Can we either pull it from something that can be verified to be the same repo that the task is coming from, or use scopes somehow, to prevent creating releases across repos?